jenkins/common/dep-build/Jenkinsfile

```diff
@@ -37,8 +37,7 @@ git submodule foreach --recursive '[ -f .git ] && echo "gitdir: $(realpath --rel
 
 make configure PLATFORM=generic
 
-make SONIC_CONFIG_BUILD_JOBS=1 target/files/stretch/ixgbe.ko \
```

Comment:
I see the module is being used by a couple for quanta system. Here is an [example](https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-quanta/ix9-32x/utils/quanta_ix9_util.py#L164). Is this being cached?